hazelcast/src/main/java/com/hazelcast/internal/bootstrap/TpcServerBootstrap.java

```diff
@@ -64,14 +68,15 @@ public class TpcServerBootstrap {
     private final boolean enabled;
     private final Map<Eventloop, Supplier<? extends ReadHandler>> readHandlerSuppliers = new HashMap<>();
     private final List<AsyncServerSocket> serverSockets = new ArrayList<>();
+    private final Config config;
     private volatile List<Integer> clientPorts;
 
     public TpcServerBootstrap(NodeEngineImpl nodeEngine) {
         this.nodeEngine = nodeEngine;
         this.logger = nodeEngine.getLogger(TpcServerBootstrap.class);
         this.ss = (InternalSerializationService) nodeEngine.getSerializationService();
-        this.enabled = Boolean.parseBoolean(getProperty("hazelcast.tpc.enabled", "false"))
-                || nodeEngine.getConfig().getAltoConfig().isEnabled();
+        this.config = nodeEngine.getConfig();
+        this.enabled = config.getAltoConfig().isEnabled();
         logger.info("TPC: " + (enabled ? "enabled" : "disabled"));
         this.writeThrough = Boolean.parseBoolean(getProperty("hazelcast.tpc.write-through", "false"));
```

Comment:
The property configuration is starting to become very inconsistent.

Reply:
https://github.com/hazelcast/hazelcast/pull/23319/commits/009609eec0df96ba2a9327217aa40ce9d50c417b